charts/incubator/trilium-notes/values.yaml

```diff
@@ -0,0 +1,19 @@
+image:
+  repository: ghcr.io/zadam/trilium
+  pullPolicy: IfNotPresent
+  tag: 0.49.4@sha256:05311b08d2cac84f1c391381879c60979233c1c424e7ab7342c780353a5e84ef
+
```

Comment:
```suggestion
  tag: v0.49.4@sha256:af932e6cd1e60f3b0c84f94c28a036c17e062d68c0d80491fc51836edad3b453

```

Reply:
Upstream tags don't contain "v" at the beginning.

Reply:
We do ;)
https://hub.docker.com/r/truecharts/trilium-notes/tags